### PR TITLE
Create wishlist page

### PR DESCRIPTION
**Pull request by Vacariuc-Alex**

### CODE REVIEW

File: src/components/InfoCard.tsx [hunk 1]
```diff
@@ -1,70 +1,60 @@
...
+            toggleFavouriteAreaVisibility("1");
         }
-    }, [elementProperties]);
+    }, [productProps]);
 
     const handleFavouriteItemWasSelected = (e: ChangeEvent<HTMLInputElement>) => {
-        if (e.target.checked) {
-            elementProperties.isFavourite = true;
+        const isChecked = e.target.checked;
+        if (isChecked) {
```
Comment (sergheistratu) [inline src/components/InfoCard.tsx]: if (e.target.checked)

not need create variable for this u can directly

File: src/pages/PageNotFound.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Navbar from "../components/Navbar";
+
+const PageNotFound = () => {
+    return(
+        <>
+            <Navbar/>
+            <br/><br/>
```
Comment (sergheistratu) [inline src/pages/PageNotFound.tsx:8]: bad practice using  tag `<br />`
use margin -top for <h1>

**Review by sergheistratu — APPROVED** [review]
fix my comments and u can continue with testing

File: src/pages/PageNotFound.tsx (hunk 2 above)
Comment (Vacariuc-Alex) [reply src/pages/PageNotFound.tsx:8]: Sure, this is already in progress in another PR (this was a temporarely modification)

File: src/components/InfoCard.tsx (hunk 1 above)
Comment (Vacariuc-Alex) [reply src/components/InfoCard.tsx]: done

**Pull request merged**